_blog.yml

```
@@ -5683,4 +5683,16 @@
     - llm
     - vlm
     - community
-    - research
\ No newline at end of file
+    - research
+
+- local: xet-on-the-hub
+  title: "Xet is on the Hub"
+  author: jsulz
+  thumbnail: /blog/assets/xet-on-the-hub/thumbnail.png
+  date: March 17, 2025
+  tags:
+    - dedupe
+    - storage
+    - content defined chunking
+    - infrastructure
+    - content addressed store
```

```suggestion
    - xet
    - infra
```
maybe an explicit xet tag instead, but we would need to update the previous posts then

---

I agree that this post has a few too many tags 😅 (don't really have to include CDC or dedupe - it's more about storage components and infrastructure). 

Not sure about xet tag 🤔 was thinking with an infra/infrastructure tag we could have the system component (e.g., "storage") dictate organization of blog content (with the `xet-team` org tag pointing people to team-specific content), but also don't have strong opinions.

For now, I just removed the unnecessary tags, but happy to make more updates!

---

yes, no strong opinion